### ISSUE TITLE
Create user login

### ISSUE THREAD

**Issue by reginafcompton** [datamade/just-spaces#8]
We can safely complete the first part of 3.1. Most of this should be out-of-the-box Django.

```
3.1 Admin backend
DataMade will setup a basic administrative interface that will allow users to create accounts, login, and manage data. The basic components will include:

* Account creation
* Secure user login
* Password retrieval
```

**Comment by reginafcompton** [comment]
This issue has one element of complexity: users should belong to a particular Agency (from the PLDP).

I can imagine a solution, wherein the users select a Agency from a dropdown (with results generated from our database). We could have UCD pre-populated as an agency, for this initial version.

We also need to think about when and how we populate our database with Agencies. Is there public form that just auto-generates these organizations? Or do all participating Agencies need to be approved by UCD?

**Comment by beamalsky** [comment]
I propose that in our first iterations we assume that DataMade/UCD will need to add new agencies

**Comment by reginafcompton** [comment]
I concur.

**Comment by reginafcompton** [comment]
We'll open a different issue for password retrieval and email set-up.

**Comment by beamalsky** [comment]
Closed by #33
